DEVELOPMENT.md

```diff
@@ -10,57 +10,67 @@ Athens uses [Go Modules](https://golang.org/cmd/go/#hdr-Modules__module_versions
 
 See our [Contributing Guide](CONTRIBUTING.md) for tips on how to submit a pull request when you are ready.
 
-### Go version
-Athens is developed on Go1.11+.
+## Go Version
 
-To point Athens to a different version of Go set the following environment variable
-```
-GO_BINARY_PATH=go1.11.X
-or whichever binary you want to use with athens
+You'll need Go version 1.11 or higher to build and run Athens. Since it uses the `go` CLI internally, you can tell it which CLI to use via the `GO_BINARY_PATH` environment. Set it to the binary of choice like this:
+
+```console
+GO_BINARY_PATH=/path/to/go
 ```
 
-# Run the Proxy
-After you've set up your dependencies, the `buffalo` CLI makes it easy to launch the proxy: 
+# Run the Proxy Natively
+
+After you've set up your Go version, the `buffalo` CLI makes it easy to launch the proxy: 
 
 ```console
 cd cmd/proxy
 buffalo dev
 ```
 
-If you don't have Buffalo installed, you can just use the `go` command directly as such: 
+If you don't have Buffalo installed, you can also use the `go` command directly like this:
 
-```
+```console
 cd cmd/proxy
 go build
 ./proxy
 ```
 
-After the server starts, you'll see some console output like:
+Whichever way you start the server, you'll see some console output like:
 
 ```console
 Starting application at 127.0.0.1:3000
 ```
 
-### Dependencies
+... and then you'll be ready to use Athens!
 
 # Services that Athens Needs
 
-Athens relies on several services (i.e. databases, etc...) to function properly. We use [Docker](http://docker.com/) images to configure and run those services. **However, Athens does not require any storage dependencies by default**. The default storage is in memory, you can opt-in to using the `fs` which would also require no dependencies. But if you'd like to test out Athens against a real storage backend (such as MongoDB, Minio, S3 etc), continue reading this section:
+By default, Athens doesn't require any external services to be running. By default, it uses in-memory storage and you can also opt into on-disk storage, and neither method requires any external services.
+
+However, Athens does support other storage drivers, and takes advantage of other external services that can be helpful for more permanent and "production-like" deployments. Here are some of the services that Athens can take advantage of:
+
+- [Minio]((https://minio.io/)) storage driver
+- [MongoDB](https://www.mongodb.com/) storage driver
+- [Jaeger](https://www.jaegertracing.io/) for distributed tracing
+
```

Comment:
We support datadog too! Are we counting this as a _service_ that Athens can take advantage of?

Reply:
Ooh good call. I added that

Reply:
Should we list other storages as well, like GCP AWS etc